From 7c0b9a8633c802c014593f6975cb60cddb0624b1 Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Sun, 9 Feb 2014 15:24:42 -0600
Subject: Fix incompatibility with automake 1.13; specify serial-tests.

As of automake 1.13, the default test harness changed to a newer one
that's not compatible with Guile 1.8's current arrangement.  Specify
serial-tests to select the old harness.

https://lists.gnu.org/archive/html/bug-automake/2013-06/msg00018.html
---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 9f23a4a..e894adc 100644
--- a/configure.in
+++ b/configure.in
@@ -41,7 +41,7 @@ AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_SRCDIR(GUILE-VERSION)
 
-AM_INIT_AUTOMAKE([gnu no-define check-news -Wall -Wno-override])
+AM_INIT_AUTOMAKE([gnu no-define check-news serial-tests -Wall -Wno-override])
 
 AC_COPYRIGHT(GUILE_CONFIGURE_COPYRIGHT)
 AC_CONFIG_SRCDIR([GUILE-VERSION])
